### PR TITLE
Rename tooling codestarts

### PR DESCRIPTION
**Pull request by ia3andy**
The goal is to avoid future conflicts between codestarts

### CODE REVIEW

File: independent-projects/tools/base-codestarts/src/main/filtered/codestarts/quarkus/tooling/gradle-wrapper/codestart.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: gradle-wrapper
+name: tooling-gradle-wrapper
```
Comment (gsmet) [inline independent-projects/tools/base-codestarts/src/main/filtered/codestarts/quarkus/tooling/gradle-wrapper/codestart.yml:1]: Maybe `core` or `quarkus-core` would be a better prefix?

But I don't mind if it stays that way. Some tests need adjustments though.

**Review by gsmet — CHANGES_REQUESTED** [review]

**Comment by ia3andy** [comment]
I was on pto, I will fix the test

**Comment by maxandersen** [comment]
aren't these names/ids relied on by past quarkus cli/plugin versions or is this purely local for a given quarkus release? i.e. does renaming these in 3.5 still make a quarkus cli 3.4 or older able to generate apps for 3.5+ ?

**Comment by ia3andy** [comment]
> aren't these names/ids relied on by past quarkus cli/plugin versions or is this purely local for a given quarkus release? i.e. does renaming these in 3.5 still make a quarkus cli 3.4 or older able to generate apps for 3.5+ ?

They are embedded in a release.

**Comment by maxandersen** [comment]
> They are embedded in a release.

okey - so these names are not exposed across versions ?

**Comment by ia3andy** [comment]
@gsmet is it me or the CI is a bit flacky these days?

File: independent-projects/tools/base-codestarts/src/main/filtered/codestarts/quarkus/tooling/gradle-wrapper/codestart.yml (hunk 1 above)
Comment (ia3andy) [reply independent-projects/tools/base-codestarts/src/main/filtered/codestarts/quarkus/tooling/gradle-wrapper/codestart.yml:1]: I would prefer to keep tooling-...

**Review by gsmet — APPROVED** [review]

**Pull request merged**